spec/forms/hyrax/forms/permission_template_form_spec.rb

```diff
@@ -195,12 +195,14 @@
 
     context "when a new workflow has been chosen" do
       it "gives the managers workflow roles" do
+        expect(form).not_to receive(:update_permission_template)
```

Comment:
@jcoyne This was to fulfill a previous request of yours:

> Is there a test that could be written that would fail before this change and would succeed after this change?